src/kernel/src/ucred/mod.rs

```diff
@@ -1,25 +1,40 @@
 pub use self::auth::*;
 pub use self::id::*;
+use self::prison::Prison;
+use self::prison::PRISON0;
 pub use self::privilege::*;
+use crate::errno::EAFNOSUPPORT;
+use crate::errno::ESRCH;
 use crate::errno::{Errno, EPERM};
+use crate::net::AddressFamily;
+use crate::ucred::prison::PrisonAllow;
+use crate::ucred::prison::PrisonFlags;
 use std::num::NonZeroI32;
 use thiserror::Error;
 
 mod auth;
 mod id;
+pub mod prison;
```

Comment:
Better to make this private and re-export instead or change the other sub-modules to public and remove re-exports.